vsts/pipelines/templates/_securityChecks.yml

```diff
@@ -33,4 +33,9 @@ steps:
   displayName: 'Post Analysis'
   inputs:
     BinSkim: true
-    CredScan: true
\ No newline at end of file
+    CredScan: true
+
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-antimalware.AntiMalware@3
```

Comment:
Doing this as part of the Security review task that I am currently working on.

Reply:
im seeing this in the log ..  what does it mean?
CmdTool: Failed with hr = 0x8024002E. Check C:\Users\VSSADM~1\AppData\Local\Temp\MpCmdRun.log for more information
##[warning]AntiMalware signature update failed with hr=8024002E (Access to an unmanaged server is not allowed.)

Reply:
Yeah, I noticed it too. Not sure if we need to worry about it, but I will contact DevOps team regarding it.